### PR TITLE
feat: onboarding 2nd iteration

### PR DESCRIPTION
**Pull request by tomaszantas**
Description
---

- [x] refactor & unify sizing of TBot components
- [x] fix fade out section (#221)
- [x] fix scroll bar spacing
- [x] fix the lack of height adaptation for single messages
- [x] use MainLayout with Onboarding
- [x] skip chatting
- [x] add blank expert view
- [x] set up flow of messages

TODO in next PRs:

- add Redux integration - installing Docker may require to reboot system, so then the user should start onboarding from different point #23
- adjust timing of messages
- styling of dynamic elements within messages (`processing...`, `done`, `fail`)
- Blockchain synchronization step
- Docker install instructions #23
- Download Docker Images #309

Motivation and Context
---

#22 
#221
#307

Partial coverage of:

#24 

How Has This Been Tested?
---

onboarding-vid1

### CODE REVIEW

**Comment by tarnas14** [comment]
the close button on help prompt got pushed out of the prompt box
image

File: applications/launchpad/gui-react/src/components/TBot/TBotPrompt/styles.ts [hunk 1]
```diff
@@ -1,68 +1,122 @@
 import styled from 'styled-components'
 import { animated } from 'react-spring'
 
+export const promptHeightSpacing = 250
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/components/TBot/TBotPrompt/styles.ts]:
```suggestion
export const PROMPT_HEIGHT_SPACING = 250
```

File: applications/launchpad/gui-react/src/components/TBot/TBotPrompt/styles.ts [hunk 2]
```diff
@@ -86,7 +140,7 @@ export const ScrollWrapper = styled.div`
 `
 
 export const MessageWrapper = styled.div`
-  padding-top: 20px;
+  // padding-top: 20px;
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/components/TBot/TBotPrompt/styles.ts]:
```suggestion
```

**Review by tarnas14 — DISMISSED** [review]

**Comment by tomaszantas** [comment]
Fixed also close button issue:

image

**Review by tarnas14 — APPROVED** [review]

**Pull request merged**